Annotato/Annotato/Persistence/AnnotationsPersistenceManager.swift

```diff
@@ -44,6 +45,26 @@ class AnnotationsPersistenceManager {
         annotation.setDeletedAt()
         return localAnnotationsPersistence.deleteAnnotation(annotation: annotation)
     }
+
+    func createOrUpdateAnnotation(annotation: Annotation) async -> Annotation? {
+        _ = await remoteAnnotationsPersistence.createOrUpdateAnnotation(annotation: annotation)
+
+        if annotation.createdAt == nil {
+            annotation.setCreatedAt()
+        }
+        annotation.setUpdatedAt()
+        return localAnnotationsPersistence.createOrUpdateAnnotation(annotation: annotation)
+    }
+
+    func createOrUpdateAnnotations(annotations: [Annotation]) async -> [Annotation]? {
+        var resultingAnnotation: [Annotation] = []
+        for annotation in annotations {
+            if let newAnnotation = await createOrUpdateAnnotation(annotation: annotation) {
+                resultingAnnotation.append(newAnnotation)
+            }
+        }
+        return resultingAnnotation
+    }
```

Comment:
This function seems to be unused now, will they be used in future?

Reply:
Nope I think it will not be used, but I implemented this because it is the plural of the function above it

Reply:
Hmm, but just because we have the singular doesn't mean we must have the plural. Let's remove it if it's not used